panel/src/components/Views/CustomView.vue

```diff
@@ -18,12 +18,12 @@ export default {
     })
   },
   watch: {
-    plugin() {
-      this.$store.dispatch("view", this.plugin);
+    plugin: {
+      handler: () => {
+        this.$store.dispatch("view", this.plugin);
+      },
+      inmediate: true
```

Comment:
There's probably a typo? Should be `immediate` (now it's `inmediate`)?

Reply:
Indeed, thanks for spotting it. Never code after talking to your Spanish-speking SO :D